packages/aws-cdk/lib/api/deployments/deployments.ts

```diff
@@ -297,23 +293,13 @@ interface AssetOptions {
 }
 
 export interface BuildStackAssetsOptions extends AssetOptions {
-  /**
-   * Options to pass on to `buildAssets()` function
-   */
-  readonly buildOptions?: BuildAssetsOptions;
-
   /**
    * Stack name this asset is for
    */
   readonly stackName?: string;
 }
 
 interface PublishStackAssetsOptions extends AssetOptions {
-  /**
-   * Options to pass on to `publishAsests()` function
-   */
-  readonly publishOptions?: Omit<PublishAssetsOptions, 'buildAssets'>;
```

Comment:
Only used by `publishSingleAsset`. None of the call sites specify `publishOptions `. Not used in `publishSingleAsset` at all.